Cargo.toml

```diff
@@ -49,6 +50,9 @@ serde_derive = "1.0.0"
 serde = "1.0.0"
 clippy = { version = "=0.0.142", optional = true }
 chrono = "0.4.0"
+pulldown-cmark = { version = "0.0.15", default-features = false }
+ammonia = "0.5.0"
+syntect = "1.7.1"
```

Comment:
Ah yes you're right, I remember reading about this behaviour in the Rust Book, even if one specifies a specific patch version it'll get updated anyway to the latest one.